js/modules.js

```diff
@@ -464,6 +464,23 @@ var StreamCloudModule = {
     }
 };
 
+var SVTOppetArkivModule = {
+  canHandleUrl: function( url ){
+    var validPatterns = [
+      ".*oppetarkiv.se/(video|klipp)/.*/.*"
+    ];
+    return urlMatchesOneOfPatterns(url, validPatterns);
+  },
+  getMediaType: function(){
+    return 'video';
+  },
+  getPluginPath: function(url, getAddOnVersion, callback) {
+      var videoId = url.match('(https|http):\/\/(www\.)?oppetarkiv\.se(\/(video|klipp)\/[0-9]+\/.*)')[3];
```

Comment:
I couldn't find any url that had just short videos aka "clips" (klipp) for openarchive as svtplay has, but i left it in there incase there was some or SVT decides to release such a thing. They seem to be running the same infrastructure, so shouldn't be a problem.